.github/scripts/ffmpeg/execute-ffmpeg-tests.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+source `dirname ${BASH_SOURCE[0]}`/../config.sh
+
+SOURCE_PATH=$(realpath `dirname ${BASH_SOURCE[0]}`/../../../code)
+skip_tests_path=$(realpath `dirname ${BASH_SOURCE[0]}`/skip_tests.txt)
+
+echo "::group::Test FFmpeg"
+    pushd $FFMPEG_TESTS_PATH
+
+    mkdir -p tests/data/fate
+    ln -s /usr/bin/base64 tests/base64
+
+    run_test=0
+    IFS=$'\n'
+    for i in $(sed '/^@/d;/fate-source/d' fate.log); do
+    if [[ $i =~ ^TEST ]]; then
+        test_name=${i##* }
+        run_test=$(cat $skip_tests_path | grep -q "^$test_name$" && echo 0 || echo 1)
+        [[ $run_test == 1 ]] && echo TEST $test_name || echo SKIP $test_name
```

Comment:
I'd say the logic in the entire for loop could be written to be more intuitively undersandable but I won't propose how, ATM.

Reply:
A small refactoring has been done and comments have been added to improve readability.